pilot/pkg/proxy/envoy/v2/cds.go

```diff
@@ -18,13 +18,14 @@ import (
 	"time"
 
 	xdsapi "github.com/envoyproxy/go-control-plane/envoy/api/v2"
+	cluster "github.com/envoyproxy/go-control-plane/envoy/config/cluster/v3"
```

Comment:
Should we move this file out of v2? like envoy/cds.go? Since now cds should be v3?

Reply:
Probably. It might make sense to move files in a different PR (could be prior to this one). I want to split this work into many smaller PRs since its really hard to review otherwise